pandas/tests/indexes/datetimes/test_datetime.py

```diff
@@ -433,3 +433,10 @@ def test_asarray_tz_aware(self):
             result = np.asarray(idx, dtype=object)
 
         tm.assert_numpy_array_equal(result, expected)
+
+    def test_apply_on_series(self):
```

Comment:
1) Could you move this test where the other timezone related tests are?

2) Test should compare a `result` output and `expected` output. In this particular case. You'll want to create 2 series and use `tm.assert_series_equal` to compare the 2.

Reply:
Thanks for the feedback, I'll get right on it :)

Reply:
I've moved the test to the "test_datetime_index" test file.

In this case I made use of assert_frame_equal as the issue is the fact that apply fails on Series that have a DateTimeIndex, and a DataFrame is not returned as expected.